vw/src/main/scala/com/microsoft/azure/synapse/ml/vw/VowpalWabbitBase.scala

```diff
@@ -75,134 +73,81 @@ trait VowpalWabbitBase extends Wrappable
 
   // can we switch to use meta programming (https://docs.scala-lang.org/overviews/macros/paradise.html)
   // to generate all the parameters?
-  val args = new Param[String](this, "args", "VW command line arguments passed")
-  setDefault(args -> "")
+  // (update: this is a removed feature as of 3.0, so would have to use newer macro mechanisms)
+  val passThroughArgs = new Param[String](this, "args", "VW command line arguments passed")
+  setDefault(passThroughArgs -> "")
+  def getPassThroughArgs: String = $(passThroughArgs)
+  def setPassThroughArgs(value: String): this.type = set(passThroughArgs, value)
 
-  def getArgs: String = $(args)
+  @deprecated("Please use 'getPassThroughArgs'.", since="1.0.0-RC3")
```

Comment:
we should deprecate this in since 0.9.6 or something as they haven't been deprecated until now